src/features/streamlabs/authorization-code.spec.ts

```diff
@@ -40,6 +37,6 @@ describe("listenForAuthorizationCode", () => {
 
     // we MUST have stopped the HTTP server since we don't need it anymore
     expect(serverMock.listening).toBe(false);
-    expect(serverClose).toHaveBeenCalledTimes(1);
+    expect(serverClose).toBeCalledTimes(1);
```

Comment:
vitest should include all the jest expects, did you cange it because you prefer the new one?

Reply:
Good catch. I did not realise I had made this unnecessary change.
Reverted in ecd1cd31670db5d0aad9b76e36c6b51cc7a4282e